pom.xml

```diff
@@ -626,12 +626,32 @@
             <activation>
                 <activeByDefault>false</activeByDefault>
             </activation>
-            <distributionManagement>
-                <site>
-                    <id>reports</id>
-                    <url>${reports.repository.url}/${version}</url>
-                </site>
-            </distributionManagement>
+            <build>
+                <plugins>
+                    <plugin>
+                        <artifactId>maven-source-plugin</artifactId>
```

Comment:
this creates an additional jar for each artifact containing the corresponding source code. Most IDEs will look for source jars for artifacts when you try to access a class in an external library and then will decompile as a last resort if no source artifact is found. Basically gives you access as a third party developer to the actual annotated source code of your dependencies